docs/docs/dev/deployments/multi-zone.md

```diff
@@ -47,6 +47,11 @@ A multi-zone deployment includes:
 * The **zone ingress**:
   * Receive XDS configuration from the local zone control plane.
   * Proxy traffic from other zone data plane proxies to local data plane proxies.
+* (optional) The **zone egress**:
+    * Receive XDS configuration from the local zone control plane.
+    * Proxy traffic from local data plane proxies:
+      * to data plane proxies from other zones;
```

Comment:
Isn't it to zone ingresses in other zones?

Reply:
It is indeed

Reply:
addressed